emergence_lib/src/organisms/life_cycles.rs

```diff
@@ -0,0 +1,23 @@
+//! Life stages are connected by life paths, creating a life cycle for each strain of organism.
+
+use bevy::{
+    prelude::Component,
+    utils::{HashMap, HashSet},
+};
+
+use super::OrganismKind;
+
+/// A collection of [`LifeStage`]s for an organism of type `O`, connected by [`LifePath`]s
+#[derive(Component, Default)]
+pub struct LifeCycle<O: OrganismKind> {
+    /// The set of all possible life cycles for an organism of type `O`.
```

Comment:
The struct is already called life cycle and we don't have recursion here
```suggestion
    /// The set of all possible life stages for an organism of type `O`.
```